Biasing/python/ecal.py

```diff
@@ -75,6 +75,45 @@ def photo_nuclear( detector, generator ) :
 
     return sim
 
+def nonfiducial_photo_nuclear( detector, generator ) :
+
+    # Instantiate the simulator. 
+    sim = simulator.simulator("photo-nuclear")
+    
+    # Set the path to the detector to use.
+    #   the second parameter says we want to include scoring planes
+    sim.setDetector( detector , True )
+    
+    # Set run parameters
+    sim.description = "ECal photo-nuclear, xsec bias 450"
+    sim.beamSpotSmear = [20., 80., 0.] #mm
+    
+    sim.generators.append( generator )
+    
+    # Enable and configure the biasing
+    #   In order to conform to older samples,
+    #   we can only attach to some of the volumes in the ecal
+    #   so we ask for this operator to be attached to the 'old_ecal'
+    sim.biasing_operators = [ bias_operators.PhotoNuclear('old_ecal',450.,2500.,only_children_of_primary = True) ]
```

Comment:
Not sure what this `old_ecal` is actually doing, or if it's correct at all, @tomeichlersmith ?

Reply:
https://github.com/LDMX-Software/SimCore/blob/191f2eccfefb20f293f9c2e8ab5e89ec3da1d9aa/src/SimCore/DetectorConstruction.cxx#L54 from three years ago lmao

I caught a very minor bug where an old method of attaching biasing operators was only biasing within the tungsten and silicon and not the other material. I suspect we can update this to `'ecal'` but that might end up changing distributions slightly since now PN is biased in more volumes.